pkg/events/eventList.go

```diff
@@ -0,0 +1,58 @@
+package events
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// List returns Events inventory in the given namespace.
+func List(
+	apiClient *clients.Settings, nsname string, options ...metaV1.ListOptions) ([]*Builder, error) {
+	if nsname == "" {
+		glog.V(100).Infof("Events 'nsname' parameter can not be empty")
+
+		return nil, fmt.Errorf("failed to list Events, 'nsname' parameter is empty")
+	}
+
+	logMessage := fmt.Sprintf("Listing Events in the namespace %s", nsname)
+	passedOptions := metaV1.ListOptions{}
+
+	if len(options) > 1 {
+		glog.V(100).Infof("'options' parameter must be empty or single-valued")
```

Comment:
Why accept a variadic parameter if you only use the first `option`?

Reply:
We want to maintain "options" as an optional argument that shouldn't exceed one.